### ISSUE TITLE
MetadataCatalog is not defined

### ISSUE THREAD

**Issue by Dicko87** [abdksyed/DETR#3]
Hi there, I am not sure how to resolve the following:

![image](https://user-images.githubusercontent.com/37139958/136788399-fd0f61c1-ee61-4616-9fae-f4f05e96997a.png)

**Comment by Dicko87** [comment]
I tried to run the following and got a picture that is a lot of rubbish and makes no sense! 
Any idea what could be wrong here?

![image](https://user-images.githubusercontent.com/37139958/136790053-e238cace-0fdb-4a0b-bea9-b0db0fad4ed2.png)

**Comment by abdksyed** [comment]
Are you using pretrained weights?
If this is random, maybe the weights are not loaded or something.

**Comment by Dicko87** [comment]
I trained the coco model from pre-trained (object detection) weights. I then loaded the best checkpoint of my panoptic model, ran an image through the model to get predictions and that is what I got, but the segmentation mask image looks good, this one does not.

I wondered if it was something to do with Detectron because when I try to import it I get the following errors:

![image](https://user-images.githubusercontent.com/37139958/136814758-eacd2840-572d-4a79-9e22-814587c87987.png)

![image](https://user-images.githubusercontent.com/37139958/136814697-6b96b5f3-0536-4010-87b1-96ac625c8ac4.png)

**Comment by Dicko87** [comment]
My process is this:
![image](https://user-images.githubusercontent.com/37139958/136815024-38a06687-1520-4907-a9db-a3b3e9a463f3.png)
which comes from here:
![image](https://user-images.githubusercontent.com/37139958/136815104-ec7e110a-c760-42fd-a40b-e82a79797fb1.png)
I then put the model in evaluation mode and feed it an image:
![image](https://user-images.githubusercontent.com/37139958/136815223-d239b3ba-cfb0-406f-bc4b-279635c0d726.png)

Then define result to be:
![image](https://user-images.githubusercontent.com/37139958/136815345-8135a47f-15ab-4aef-83f9-5535b0ec9a9c.png)

Then use it to give me the png image which is rubbish.
![image](https://user-images.githubusercontent.com/37139958/136815439-e3a8c294-9e9b-4493-9327-f1063918fd8a.png)

**Comment by abdksyed** [comment]
can you keep ```strict=True``` in your function, because than we will know, if the weights and the model are matching or not.

**Comment by Dicko87** [comment]
Thank you. I tried that and does seem to help but it is not as good as I would like it.
As you can see, the image below nicely shows two yellow sticks which is what I want but the png just isnt quite as good.
![image](https://user-images.githubusercontent.com/37139958/136818431-b48a1b15-33ed-41fa-b865-65c4c941c4d9.png)

![image](https://user-images.githubusercontent.com/37139958/136818495-b648ef3f-396e-4ab5-9c28-898fe79c0e98.png)

**Comment by Dicko87** [comment]
I also think something isn't quite right because there are two detections for example in the above mask and what is actually printed are two identical pictures rather than one for each mask.

![image](https://user-images.githubusercontent.com/37139958/136819671-a796842a-d1dd-4a70-afef-b7dcf8f0175b.png)

As you can see I have two outputs/predictions with slightly different numbers:
![image](https://user-images.githubusercontent.com/37139958/136820288-2a7d746d-c21a-44db-a4eb-c5ba7827d580.png)

![image](https://user-images.githubusercontent.com/37139958/136820174-e41e7f4d-3eaf-4b49-a067-e7961fe805e2.png)
![image](https://user-images.githubusercontent.com/37139958/136820218-96050e9e-34eb-421f-ade2-5acd1d40b6f6.png)

**Comment by abdksyed** [comment]
I think it's detecting the bounding box as combined, can you plot the ```output['pred_boxes']``` on the image, and see what is the bounding box, if the two bounding box are separate than mask is wrong, but if the bounding box itself is around both the objects, than the mask is correct.

**Comment by Dicko87** [comment]
I plotted the bounding boxes and they are perfect. They are around the two sticks perfectly, so there must be something up with the masks then?
